AutomatedTesting/Gem/PythonTests/Prefab/tests/add_entity/AddEntity_UnderPrefab.py

```diff
@@ -0,0 +1,53 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+def AddEntity_UnderPrefab():
+    """
+    Test description:
+    - Creates an entity.
+    - Creates a prefab out of the above entity.
+    - Focuses on the created prefab and adds a new child entity.
+    Checks that the entity is correctly added.
```

Comment:
nit: missing "-"

Reply:
Addressed feedback with those 3 tests in latest commit.